src-input/duk_strings.h

```diff
@@ -61,7 +62,6 @@
 #define DUK_STR_DECODE_FAILED                    "decode failed"
```

Comment:
Since this string will now be used by TextDecoder and not only internally in e.g. `duk_compile()`, maybe it could be clarified: "text decoding error".  Then there could also be a verbose version that includes the name of the encoding, e.g. "unable to decode text as utf-8"